app/src/services/subscription.service.js

```diff
@@ -120,6 +124,14 @@ class SubscriptionsService {
                 _id: { $nin: [area.id, area.subscriptionId] }
             });
 
+            if (areas && areas.length > 0) area.status = 'saved';
```

Comment:
this seems unrelated to the scope of this PR

Reply:
We update the status of an area to 'saved' if at least one area with the same `geostore` already has status 'saved' in the DB. We want to keep this behavior for the newly added `geostoreDataApi` field, since the behavior should be exactly the same as with the `geostore` field